### ISSUE TITLE
Add Common functions to juliamind.nn.functional.common

### ISSUE THREAD

**Issue by diksha-shrivastava13** [diksha-shrivastava13/JuliaMind#2]
> Please create a new issue with each of the subtasks and comment "- [ ] #issue_number" to keep the task list updated.

- [ ] unfold
- [ ] interpolate
- [ ] \_is_list_or_turple_
- [ ] upsample
- [ ] bilinear
- [ ] dropout
- [ ] get_attrs
- [ ] dropout2d
- [ ] dropout3d
- [ ] alpha_dropout
- [ ] pad
- [ ] zeropad2d
- [ ] cosine_similarity
- [ ] linear
- [ ] quant_for_compress
- [ ] linear_compress
- [ ] label_smooth
- [ ] class_center_sample
- [ ] fold
- [ ] \_is_list_or_turple_
